### PR TITLE
New package: zookeeper-3.4.10

### PR DESCRIPTION
**Pull request by Morsicus**
Hi, 

When zookeeper will be present in our repository i can start a wiki page if it's ok.

Bye

### CODE REVIEW

File: srcpkgs/zookeeper/files/conf/environment [hunk 1]
```diff
@@ -0,0 +1,3 @@
+tickTime=2000
+dataDir=/var/zookeeper
+clientPort=2181
```
Comment (Vaelatern) [inline srcpkgs/zookeeper/files/conf/environment]: Can't this be pulled from the repository?

File: srcpkgs/zookeeper/files/zookeeper/run [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+DEFAULT_CONF_PATH='/etc/zookeeper/conf/environment'
+
+/usr/share/zookeeper/bin/zkServer.sh start-foreground ${DEFAULT_CONF_PATH}
```
Comment (Vaelatern) [inline srcpkgs/zookeeper/files/zookeeper/run]: Please read other runit scripts. This does not meet our requirements, basically needs `exec` and can't it possibly drop privileges?

File: srcpkgs/zookeeper/files/conf/environment (hunk 1 above)
Comment (Morsicus) [reply srcpkgs/zookeeper/files/conf/environment]: Done

File: srcpkgs/zookeeper/files/zookeeper/run (hunk 2 above)
Comment (Morsicus) [reply srcpkgs/zookeeper/files/zookeeper/run]: I forgot 'exec' … Added. 

Checking for privileges.

**Comment by the-maldridge** [comment]
This should be able to run as a non-root service user.  Please refactor to include this change.

File: srcpkgs/zookeeper/template [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+revision=1
+noarch=yes
+short_desc="A dristributed hierarchical key-value store"
+maintainer="Mickael Fortunato <morsi.morsicus@gmail.com>"
+license="Apache-2.0"
+homepage="http://zookeeper.apache.org/"
+distfiles="http://mirrors.ircam.fr/pub/apache/zookeeper/stable/zookeeper-${version}.tar.gz"
+checksum="7f7f5414e044ac11fee2a1e0bc225469f51fb0cdf821e67df762a43098223f27"
+depends="virtual?java-runtime"
+
```
Comment (Vaelatern) [inline srcpkgs/zookeeper/template]: Add a `system_accounts="_zookeeper"`

File: srcpkgs/zookeeper/files/zookeeper/run [hunk 4]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+DEFAULT_CONF_PATH='/etc/zookeeper/conf/environment'
+
+exec /usr/share/zookeeper/bin/zkServer.sh start-foreground ${DEFAULT_CONF_PATH}
```
Comment (Vaelatern) [inline srcpkgs/zookeeper/files/zookeeper/run]: `exec chpst -u _zookeeper /usr/share/zookeeper/bin/zkServer.sh start-foreground ${DEFAULT_CONF_PATH}`

File: srcpkgs/zookeeper/files/zookeeper/run [hunk 5]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+DEFAULT_CONF_PATH='/etc/zookeeper/conf/environment'
```
Comment (Vaelatern) [inline srcpkgs/zookeeper/files/zookeeper/run]:
```
[ -f ./conf ] && . ./conf
: ${DEFAULT_CONF_PATH:='/etc/zookeeper/conf/environment'}
```

Comment (Morsicus) [reply srcpkgs/zookeeper/files/zookeeper/run]: I'll try to work on this PR tomorrow or this week end :). Thanks for your reviews

Comment (Morsicus) [reply srcpkgs/zookeeper/files/zookeeper/run]: Done

File: srcpkgs/zookeeper/files/zookeeper/run (hunk 4 above)
Comment (Morsicus) [reply srcpkgs/zookeeper/files/zookeeper/run]: Done

**Comment by Morsicus** [comment]
Any news ?

File: srcpkgs/zookeeper/template [hunk 6]
```diff
@@ -0,0 +1,46 @@
+# Template file for 'zookeeper'
+
```
Comment (the-maldridge) [inline srcpkgs/zookeeper/template]: This space shouldn't be here.

**Comment by the-maldridge** [comment]
I chose to just fix it in the merge so I could go to bed faster.  Thanks for bearing with us during the longer than usual review.

**Pull request closed without merging**